spec/models/casa_org_spec.rb

```diff
@@ -128,52 +134,61 @@
       end
     end
 
-    describe "mileage rate for a given date" do
-      let(:casa_org) { build(:casa_org) }
+    describe "generates default contact types" do
+      let(:contact_topics) { CaseContactTopic.where(casa_org: org).pluck(:title) }
 
-      describe "with a casa org with no rates" do
-        it "is nil" do
-          expect(casa_org.mileage_rate_for_given_date(Date.today)).to be_nil
-        end
+      it "matches default contact topics" do
+        expected = fake_topics.map { |topic| topic["title"] }
+        expect(contact_topics).to include(*expected)
       end
+    end
+  end
 
-      describe "with a casa org with inactive dates" do
-        let!(:mileage_rates) do
-          [
-            create(:mileage_rate, casa_org: casa_org, effective_date: 10.days.ago, is_active: false),
-            create(:mileage_rate, casa_org: casa_org, effective_date: 3.days.ago, is_active: false)
-          ]
-        end
-
-        it "is nil" do
-          expect(casa_org.mileage_rates.count).to eq 2
-          expect(casa_org.mileage_rate_for_given_date(Date.today)).to be_nil
-        end
+  describe "mileage rate for a given date" do
+    let(:casa_org) { build(:casa_org) }
+
+    describe "with a casa org with no rates" do
+      it "is nil" do
+        expect(casa_org.mileage_rate_for_given_date(Date.today)).to be_nil
       end
+    end
```

Comment:
I corrected the nesting here, which resulted in a massive amount of diff in this file. I promise nothing really changed.